src/v2/Apps/Components/AppShell.tsx

```diff
@@ -57,6 +57,25 @@ export const AppShell: React.FC<AppShellProps> = props => {
     document.body.setAttribute("data-test", "AppReady")
   }, [])
 
+  useEffect(() => {
+    let script: HTMLScriptElement
+    if (match?.location.pathname === "order/success") {
+      script = document.createElement("script")
+
+      script.id = "mntn_conversion"
+      script.src = "../../System/Analytics/MNTN"
+      script.async = true
+      script.type = "javascript/text"
+
+      document.body.appendChild(script)
+    }
+    return () => {
+      if (document.getElementById("mntn_conversion")) {
+        document.body.removeChild(script)
+      }
+    }
+  }, [match])
```

Comment:
Nice! I actually had a `useScript` hook written to the same effect... then began to wonder if I could use a hook inside a useEffect, because the dependency array isn't linked to the `src` changing, but the `match`. 

I'll check out the hook you've mentioned and see if this question even remains relevant.